tests/unit/s2n_record_test.c

```diff
@@ -185,7 +187,7 @@ int main(int argc, char **argv)
         EXPECT_SUCCESS(s2n_stuffer_wipe(&conn->in));
         EXPECT_SUCCESS(s2n_stuffer_reread(&conn->out));
         EXPECT_SUCCESS(s2n_stuffer_copy(&conn->out, &conn->in, s2n_stuffer_data_available(&conn->out)));
-        EXPECT_FAILURE(s2n_record_parse(conn));
+        EXPECT_FAILURE(conn->secure.cipher_suite->record_alg->record_parse(conn));
```

Comment:
There's a lot of these identical changes. Would it make sense to just make s2n_record_parse() a macro that expands to conn->secure.cipher_suite->record_alg->record_parse